app/components/CvHeader/index.js

```diff
@@ -15,9 +15,10 @@ function CvHeader({ user }) {
           round />
         <div className={styles.contact}>
           <div>{user.firstName} {user.lastName}</div>
+          <div>{user.position} @ Studs</div>
           <div>Studying {user.master} @ KTH</div>
-          <div>{user.email}</div>
-          <div>{user.phone}</div>
+          <div><a href={'mailto:' + user.email}>{user.email}</a></div>
```

Comment:
Do we need the divs here now? Do they add any specific style?

We can also use template strings here instead of concatenation.

Reply:
They add line breaks but that's all.

I will change to template strings!